test/uk/gov/hmrc/helptosavefrontend/forms/EmailValidationSpec.scala

```diff
@@ -16,18 +16,21 @@
 
 package uk.gov.hmrc.helptosavefrontend.forms
 
+import cats.data.NonEmptyList
+import cats.data.Validated.Valid
```

Comment:
are we using these imports ?

Reply:
Valid is being used, the other isn't so I have removed it